resources/idlharness.js

```diff
@@ -2274,6 +2285,11 @@ IdlInterface.prototype.test_interface_of = function(desc, obj, exception, expect
 IdlInterface.prototype.has_stringifier = function()
 //@{
 {
+    if (this.name == "DOMException")
+    {
+        // Stringifier is inherited from Error
```

Comment:
Check if anyone checks for `hasOwnProperty("toString")` if you return true here; that'd be wrong.